### ISSUE TITLE
[BUG]If first element inside .nav-sidebar is a li.nav-header the padding is different to other li.nav-header

### ISSUE THREAD

**Issue by pauldbentley** [ColorlibHQ/AdminLTE#3409]
**Describe the bug**
If first element inside .nav-sidebar is a li.nav-header the padding is different to other li.nav-header.  I often like to use a .nav-header as the first item.

**To Reproduce**
1. Add `<li class="nav-header">TEST</li>` as the first item in the .nav-sidebar

**Expected behavior**
The padding to the left/right should be of the li.nav-header should be 1rem as per all subsequent headers

**Screenshots**
![nav-header](https://user-images.githubusercontent.com/5243982/107854429-a238d680-6e13-11eb-8a4d-1c9f47059059.png)

**Environment (please complete the following information):**
- AdminLTE Version: 3.1.0-rc
- Operating System:  Windows 10
- Browser (Version): Chrome (latest)

**Additional context**
The problem seems to be in [_main-sidebar.scss](https://github.com/ColorlibHQ/AdminLTE/blob/582f958c5b7b6e152279039ec855b194728dfc93/build/scss/_main-sidebar.scss#L182)

```css
.nav-header {
    font-size: .9rem;
    padding: $nav-link-padding-y;

    &:not(:first-of-type) {
      padding: 1.7rem 1rem .5rem;
    }
  }
```

So the .nav-header only gets the 1rem for left and right if it is not the first-of-type.

**Comment by danny007in** [comment]
COOL 😄 
create PR for this 👍🏼

**Comment by danny007in** [comment]
After that add compared screenshot

**Comment by danny007in** [comment]
better create an modifier `nav-main-header`

**Comment by pauldbentley** [comment]
Would it need a separate class? Should all headers have the same padding,
unless it’s not the first one where they get more top padding?

**Comment by danny007in** [comment]
What if?
if 2 header required at starting

**Comment by danny007in** [comment]
or header needed to default padding same in starting also

**Comment by pauldbentley** [comment]
For my usecase I only need 1 header at the start.  I'm happy to provide PR for this, and then if support for 2 headers at the start is requested this could be looked at.

**Comment by REJack** [comment]
I see no relevance in the changed padding for the first header element, I remove it and fix the alignment from the nav header's.

**Comment by REJack** [comment]
Fixed with 378a39ea125c216004ac4127f64260f21d3a1932.

**Comment by pauldbentley** [comment]
Thanks @REJack that fixes the issue.
